src/main/java/ToDo.java

```diff
@@ -0,0 +1,12 @@
+public class ToDo extends Task{
+
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString(){
+        return "[T]" + super.toString();
+    }
+}
```

Comment:
overall, code is very well written and follows coding standard for the most part. great job!